src/prefect/settings/models/server/services.py

```diff
@@ -385,6 +385,13 @@ class ServerServicesSchedulerSettings(ServicesBaseSetting):
         ),
     )
 
+    recent_deployments_loop_seconds: float = Field(
+        default=5,
+        description="""
+        The number of seconds the recent deployments scheduler will wait between checking for recently updated deployments. Defaults to `5`.
+        """,
+    )
+
```

Comment:
I think its overkill to define a separate subclass for the recent deployments scheduler since its a thin subclass of `Scheduler`